github-actions/oss-readme-template/src/readme.ts

```diff
@@ -146,6 +163,7 @@ export function generateReadmeStringFromTemplateString(
   const templateContext: ReadmeTemplateContext = {
     ossHeader: ossHeader,
     ossFooter: ossFooter,
+    ossUsageExamplePath: ossUsageExamplePath,
```

Comment:
Then here, you would do something like this:

```
usageExampleCode: usageExampleCode
```

And in the test template, you'd add a `{{ usageExampleCode }}` somewhere in the `Usage` section.